docs/concepts/managed-resources.md

```diff
@@ -108,21 +268,33 @@ source of truth for the external resource. This means that if someone changed a
 configuration in the UI of the provider, like AWS Console, Crossplane will
 change it back to what's given under `spec`.
 
+#### Connection Details
+
+Some Crossplane resources support writing connection details - things like URLs,
+usernames, endpoints, and passwords to a Kubernetes `Secret`. You can specify
+the secret to write by setting the `spec.writeConnectionSecretToRef` field. Note
+that while all managed resources have a `writeConnectionSecretToRef` field, not
+all managed resources actually have connection details to write - many will
+write an empty `Secret`.
+
+> Which managed resources have connection details and what connection details
```

Comment:
This seems quite subjective. Personally I prefer the note to 'interrupt' me closer to where it's relevant, rather than being a footnote at the end of the document that I may never encounter if I'm opening a heading to get some specific targeted context about how something works rather than reading the entire document end-to-end.